src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
+    function App() {
```

Comment:
add an extra blank space between imports and the function to distinguish the code a bit more. Fix indentation for function with the starting line at the beginning of the line. lines 9 and 10 should also be indented one space so closing div tag matches indentation and TodoList tag matches the rest of the components in the div.